about.html

```diff
@@ -0,0 +1,175 @@
+    <link rel="stylesheet" href="css/desktop.css" />
+    <title>Capstone Project1</title>
+  </head>
+  <body>
+    <header class="desktop-nav-header">
+      <div class="top-nav">
+        <a href="#" class="top-nav-link">
+          <i class="fa-brands fa-facebook-f"></i>
+        </a>
+        <a href="#" class="top-nav-link">
+          <i class="fa-brands fa-twitter"></i>
+        </a>
+        <a href="#" class="top-nav-link"> English </a>
+        <a href="#" class="top-nav-link"> myPage </a>
+        <a href="#" class="top-nav-link last"> logout </a>
+      </div>
+      <nav class="desktop">
+        <div class="desktop-logo">
+          <a href="index.html" class="desk-logo">
+            <img src="images/logo.png" alt="logo" />
+          </a>
+        </div>
+        <div class="desktop-nav">
+          <ul class="desktop-nav-lists">
+            <li class="nav-list">
+              <a class="nav-link" href="about.html">About</a>
+            </li>
+            <li class="nav-list">
+              <a class="nav-link" href="#program">Program</a>
+            </li>
+            <li class="nav-list">
+              <a class="nav-link" href="index.html">Speakers</a>
+            </li>
+            <li class="nav-list">
+              <a class="nav-link" href="index.html#Sponsors">Sponsor</a>
+            </li>
+            <li class="nav-list">
+              <a href="">
+                <button class="nav-btn btn">Campaign</button>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <!-- Hero section -->
+    <section class="hero" id="home">
+      <button class="nav-toggle" aria-label="toggle-navigation">
+        <!-- <span class="hamburger"></span> -->
+        <div class="hamburger"></div>
+      </button>
+      <nav class="mobile-nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="index.html" class="nav-link">Home</a>
+          </li>
+          <li class="nav-item">
+            <a href="about.html" class="nav-link">about</a>
+          </li>
+          <li class="nav-item">
+            <a href="#program" class="nav-link">program</a>
+          </li>
+          <li class="nav-item">
+            <a href="#speakers" class="nav-link">speakers</a>
+          </li>
+        </ul>
+        <a href="">
+          <button class="nav-btn btn">CC Compaign</button>
+        </a>
+      </nav>
+      <div class="hero-body container">
+        <h2 class="hero-med-title">"Hello! Rugby fans"</h2>
+        <h1 class="hero-main-title">
+          Welcome to the home of Cape Town Rugby 10's
+        </h1>
+        <p class="hero-subtitle">
+          A fun festival where people from all around the world come together to
+          support their teams.
+        </p>
+        <h3 class="venue">3 – 4 FEBRUARY 2023 | ENTRIES OPEN 1 APRIL 2022</h3>
+      </div>
+    </section>
+
+    <section class="about-section">
```

Comment:
- [ ] Kindly ensure that your about-page design matches the design guidelines given. Please take note of the background, typography, color, padding/margins 👍🏽 

| Design Guideline  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/190247420-b86f91d4-b5c6-4370-8b0d-e6a2a2f4fd45.png))  | ![image](https://user-images.githubusercontent.com/63473584/190247562-ea533213-95a0-4cf8-94e6-f0801718d3a2.png)  |
| ![image](https://user-images.githubusercontent.com/63473584/190247287-f9ee1621-15e1-4883-bb4e-85f20afea557.png)  | ![image](https://user-images.githubusercontent.com/63473584/190247665-b5f3dc61-b6ab-4f47-bdfa-44c960ad4e0a.png)  |